src/markdown/setup/connection-string.md

```diff
@@ -16,28 +16,29 @@
+| `refreshSchema`            | (true/false) If true, generates (refreshes) the SQL schema with each connection. It creates a new version, leaving any existing versions in place. _Caution: use only when necessary to update schema as it can adversely affect performance._                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | `false`
+| `defaultAuthDb`            | (string) The name of the authentication database to use when authenticating with the passed `user` and `password`. This is where the authorized user is stored and can be different from what databases the user may have access to. On Amazon DocumentDB, all users are attributed to the `admin` database.                                                                                                                                                                                                                                                                                                                                                                                                                                | `admin`
+| `allowDiskUse`             | (enum/string) Provides the option for using the `allowDiskUse` query option globally on all queries. A value of `default` indicates that the system setting should be used. A value of `enable` or `disable` sets the option to `true` or `false`, respectively, on the query option for all queries.                                                                                                                                                                                                                                                                                                                                                                                                                                       | `enable`
```

Comment:
missing the alternative to set this.
Will we document how the custom_options env works?